### PR TITLE
PR 10 add rename album

### PR DESCRIPTION
**Pull request by ACEof**
Add the possibility to rename albums.
There is a functional from PR12 here, because the branch was created from PR 12.

### CODE REVIEW

File: migrations/20180716062802-add-userID-into-albums.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  up: (queryInterface, Sequelize) => {
+    queryInterface.addColumn(
+      'albums',
+      'userID',
+      Sequelize.INTEGER
+    )
+  },
+
+  down: (queryInterface, Sequelize) => {
+    /*
```
Comment (abtv) [inline migrations/20180716062802-add-userID-into-albums.js]: remove the comments if you don't plan to go to previous db schema

File: models/users.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  });
+
+  const findUserId = async (userName) => {
+    const select = await db.query('SELECT id from users where username = ?', 
+       { replacements: [userName], type: db.QueryTypes.SELECT})  
+     return select;  
+ }
+
+ function findOrCreateUser(name){
+  Users
```
Comment (abtv) [inline models/users.js]: do you need a `return` here? usually find returns something

File: models/albums.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+const db = require('./index');
+
+const Albums = db.define('albums', {
+  id: { type: Sequelize.INTEGER, primaryKey: true, autoIncrement: true, allowNull: false },
+  albumtitle: Sequelize.STRING,
+  user_id: Sequelize.INTEGER
+});
+
+function createAlbum(title, id) {
+  Albums
```
Comment (abtv) [inline models/albums.js]: should you wait when it finish?

File: models/users.js (hunk 2 above)
Comment (ACEof) [reply models/users.js]: Don't need return)

File: migrations/20180716062802-add-userID-into-albums.js (hunk 1 above)
Comment (ACEof) [reply migrations/20180716062802-add-userID-into-albums.js]: Ok. I delete. Commets was created sequelize migration.

File: models/albums.js (hunk 3 above)
Comment (ACEof) [reply models/albums.js]: Hmm. It's good question. I need to think. When i tested there was no problem.

File: models/users.js (hunk 2 above)
Comment (ACEof) [reply models/users.js]: Need return. If not use `return`, albums page don't render.

**Pull request merged**